corehq/apps/dump_reload/tests/test_dump_models.py

```diff
@@ -51,6 +51,10 @@
     "notifications.DismissedUINotify",
     "notifications.LastSeenNotification",
     "notifications.Notification",
+    "otp_static.StaticDevice",
+    "otp_static.StaticToken",
+    "otp_totp.TOTPDevice",
+    "saved_reports.ScheduledReportsCheckpoint",
```

Comment:
@dannyroberts  Can you confirm that this is correct - if we moved a domain between servers, we would **not** want to include ScheduledReportsCheckpoint info in the dump?

Reply:
ping

Reply:
I don't see why we would want to avoid this model. My read of it suggests that if we did not port this information over, then we could send out the same scheduled report multiple times to the same user. I don't think the reverse is true -- if we moved this data over, we wouldn't be risking *not* sending out the report to that user at all, because I'd assume the celery tasks handling it would still continue.

Reply:
Cool, thanks. Added ff0b4caed01 to add it - I'm just adding all rows in that table, since it looks like it's system-wide data, not specific to a domain.